src/main/java/com/onaio/steps/activities/HouseholdSummaryActivity.java

```diff
@@ -0,0 +1,78 @@
+package com.onaio.steps.activities;
+
+import android.app.Activity;
+import android.graphics.drawable.ColorDrawable;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.ImageView;
+import android.widget.LinearLayout;
+import android.widget.TextView;
+
+import com.onaio.steps.R;
+import com.onaio.steps.helper.DatabaseHelper;
+import com.onaio.steps.model.Household;
+import com.onaio.steps.model.InterviewStatus;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+
+public class HouseholdSummaryActivity extends Activity {
+
+    private List<SummaryItem> summaryItemList;
+
+    @Override
+    protected void onCreate(Bundle savedInstanceState) {
+        super.onCreate(savedInstanceState);
+        setContentView(R.layout.activity_household_summary);
+        getActionBar().setDisplayHomeAsUpEnabled(true);
+        getActionBar().setIcon(new ColorDrawable(getResources().getColor(android.R.color.transparent)));
```

Comment:
You can alternatively use `getActionBar().setDisplayShowHomeEnabled(false);`. This is still OK

Reply:
done.

Reply:
Confirmed